backend/src/main/java/com/javahelp/backend/domain/user/register/client/ClientRegisterInteractor.java

```diff
@@ -0,0 +1,81 @@
+package com.javahelp.backend.domain.user.register.client;
+
+import com.javahelp.backend.data.IUserStore;
+import com.javahelp.backend.domain.user.register.RegisterUserResponse;
+import com.javahelp.backend.domain.user.register.UserRegisterInteractor;
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.User;
+import com.javahelp.model.user.UserPassword;
+
+/**
+ * An interactor that creates new {@link User} and {@link UserPassword} for this client
+ */
+public class ClientRegisterInteractor extends
+        UserRegisterInteractor<IClientRegisterInputBoundary, ClientUserInfo> {
+
+    IUserStore userStore;
```

Comment:
There is no need to duplicate this field, the superclass already has it. Putting it here again hides the superclass field.